coordinator/src/node/resize.rs

```diff
@@ -98,7 +98,17 @@ impl Node {
             },
         )?;
 
-        db::positions::Position::set_open_position_to_resizing(conn, position.trader.to_string())?;
+        let pool = self.pool.clone();
+        tokio::spawn({
```

Comment:
Interestingly, I wasn't able to reproduce it, although I was using `std::thread::spawn` instead.